HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -99,20 +138,33 @@ def __init__(self, cFunc=None, vFunc=None,
         Returns
         -------
         None
-        '''
+        """
         # Change any missing function inputs to NullFunc
         self.cFunc = cFunc if cFunc is not None else NullFunc()
         self.vFunc = vFunc if vFunc is not None else NullFunc()
         self.vPfunc = vPfunc if vPfunc is not None else NullFunc()
         # vPFunc = NullFunc() if vPfunc is None else vPfunc
         self.vPPfunc = vPPfunc if vPPfunc is not None else NullFunc()
-        self.mNrmMin      = mNrmMin
-        self.hNrm         = hNrm
-        self.MPCmin       = MPCmin
-        self.MPCmax       = MPCmax
+        self.mNrmMin = mNrmMin
+        self.hNrm = hNrm
+        self.MPCmin = MPCmin
+        self.MPCmax = MPCmax
 
-    def appendSolution(self,new_solution):
-        '''
+    def __eq__(self, solution: HARKobject) -> bool:
```

Comment:
This should ideally be present in the parent class `HARKobject` (similarly for `AgentType`).

Reply:
This makes sense, although I put them in PerfForesight Model and Solution since these implementations might not be ready for generic models and generic solutions. Do you think I should proceed to moving them to HARKobject and AgentType as is? or maybe generic `__eq__` comparison should be on another PR?

Reply:
Yeah, let's do that. Can you remove the __eq__ tests from this PR and move it to another branch?

Reply:
removed

Reply:
I have removed everything related to `__eq__` and will do this in own PR